greedybear/celery.py

```diff
@@ -10,6 +10,8 @@
 from django.conf import settings
 from kombu import Exchange, Queue
 
+from greedybear.settings import DB_BACKUP_FREQUENCY
```

Comment:
no need, we already import `from django.conf import settings`

Reply:
Ok Sir